src/status_im/ui/screens/syncing/sheets/enter_password/views.cljs

```diff
@@ -0,0 +1,88 @@
+(ns status-im.ui.screens.syncing.sheets.enter-password.views
+  (:require [quo.react-native :as rn]
+            [status-im.ui.screens.syncing.sheets.enter-password.styles :as styles]
+            [re-frame.core :as re-frame]
+            [quo.core :as quo]
+            [status-im.i18n.i18n :as i18n]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.components.toolbar :as toolbar]
+            [clojure.string :as string]
+            [taoensso.timbre :as log]
+            [status-im.utils.security :as security]
+            [status-im.ethereum.core :as ethereum]
+            [status-im.utils.fx :as fx]
+            [status-im.native-module.core :as status]
+            [quo2.components.info.information-box :as information-box]
+            [status-im.ui.components.qr-code-viewer.views :as qr-code-viewer]
+            [status-im.utils.handlers :refer [>evt]]))
+
+(def entered-password (atom ""))
```

Comment:
how come we are not declaring these atoms as let variables in the component?

Reply:
In this specific case I want the entered-password to be available to be consumed by other functions in the call hence I have hoisted the state up 

you can see its usage here : https://github.com/status-im/status-mobile/blob/915b83c0c7aca16e9f21cb2245e7b282c6042a5b/src/status_im/ui/screens/syncing/sheets/enter_password/views.cljs#L58

I could also have stored this in app-db but I feel like that would have been an overkill :)

Reply:
no more top level stateful vars :)